packages/expo-notifications/src/InstallationIdProvider.web.ts

```diff
@@ -0,0 +1,21 @@
+import UUID from 'uuid-js';
+
+import { InstallationIdProvider } from './InstallationIdProvider';
+
+const INSTALLATION_ID_KEY = '@@expo-notifications.InstallationId@@';
```

Comment:
@EvanBacon We need to have a naming convention for Expo-internal keys in shared namespaces (like localStorage), what do you propose?

Reply:
Sorry for using this as the first proposal, I had `nativeEmitterSubscriptionKey` of `@unimodules/core/EventEmitter` in memory at that time which is [`@@nativeEmitterSubscription@@`](https://github.com/expo/expo/blob/3e7135a3d61f87e6a6cf6be7c258069633a8c104/packages/@unimodules/react-native-adapter/src/EventEmitter.ts#L4).

I went through `localStorage` usages in `expo/packages` and they are:
- `EXPO_CONSTANTS_INSTALLATION_ID` for `expo-constants.installationId`
- `EXPO_ERROR_RECOVERY_STORAGE` for `expo-error-recovery.recoveryProps`

so I'll go with

- `EXPO_NOTIFICATIONS_INSTALLATION_ID` for `expo-notifications.installationId`

🙂